Integrations/integration-Quinc.yml

```diff
@@ -0,0 +1,341 @@
+        try:
+            res = requests.request(method, url, data=body, headers=headers, params=params, verify=USE_SSL)
+            res.raise_for_status()
+        except HTTPError, e:
+            err_json = res.json()
+            err_msg = ''
+            if 'message' in err_json:
+              err_msg = err_msg + 'Error: {0}.\n'.format(err_json['message'])
+            elif 'http_response' in err_json:
+              err_msg = err_msg + 'Error: {0}.\n'.format(err_json['http_response'])
+            if 'code' in err_json:
+              err_msg = err_msg + 'QRadar Error Code: {0}'.format(err_json['code'])
+            return_error(err_msg)
+        return res.json()
+
+    def SendQuincTestRequest():
+        url = 'api/v2/enterpriseapi/statuscheck';
+        res = send_request('GET', QUINC_URL + url)
+        return res.lower()
+
+    def QuincLegacyAgentSoftwareInventory(arg):
+        url = 'api/v2/enterpriseapi/agent/'+ arg['caseid'] +'/sofwareinventory';
+        data = {
+            'ips': { 'targets' : [arg['targetIP']]},
+            'SoftwareInvJob': {'Operation': 24 }
+        }
+        res = send_request('POST', QUINC_URL + url, json.dumps(data))
+        executeCommand('delContext', { key: JOBID_KEY });
+        return {
+            'Type': entryTypes['note'],
+            'Contents': res,
+            'ContentsFormat': formats['text'],
+            'HumanReadable': 'JobID: ' + str(res),
+            'EntryContext': {JOBID_KEY : res}
+        }
+
+    def create_jobstate_context(contents):
+        return {
+             JOB_KEY+'(val.ID == obj.ID)' : contents
+        }
+
+    def create_contents(id, state, result):
+        ec = {
+            'ID' : id,
+            'State': state,
+            'Result' : result
+        }
+        return ec
+
+    def quinc_get_jobstatus_command(args):
+        url = 'api/v2/enterpriseapi/core/' + str(args['caseID']) + '/getjobstatus/' + str(args['jobID'])
+        res = send_request('GET', QUINC_URL + url)
+
+        if 'resultData' not in res:
+            contents = create_contents(args['jobID'], 'Failed', res)
+            return {
+                'Type' : entryTypes['note'],
+                'ContentsFormat' : formats['json'],
+                'Contents': contents,
+                'HumanReadable': "No data in job result",
+                'EntryContext': create_jobstate_context(contents)
+            }
+        resultData = json.loads(res['resultData'])
+        if 'RealData' not in resultData:
+            contents = create_contents(args['jobID'], 'Failed', resultData)
+            return {
+                'Type' : entryTypes['note'],
+                'ContentsFormat' : formats['json'],
+                'Contents': contents,
+                'HumanReadable': "No data in job result",
+                'EntryContext': create_jobstate_context(contents)
+            }
+        RealData = json.loads(resultData['RealData'])
```

Comment:
RealData -> real_data
resultData -> result_data

according to https://www.python.org/dev/peps/pep-0008/